### ISSUE TITLE
[Staging & Production] Figshare: Unclear file path on File Detail page

### ISSUE THREAD

**Issue by nicipfeiffer** [CenterForOpenScience/osf.io#2221]
## Steps
1. Go to a project's settings page and authorize the figshare add-on
2. Link a folder to the project
3. Go to the files tab and select a file to view

## Expected
On the files detail page for a figshare file the file path on the right side of the page should give more useful information.
That figshare would have the same structure to the filepath as with other add-ons.

![screen shot 2015-03-18 at 8 42 30 am](https://cloud.githubusercontent.com/assets/9381062/6709389/5e83492c-cd4f-11e4-8e71-b9455e6d63c8.png)

## Actual
When other add-on files are viewed on the file detail page the file path gives the project/ add-on/ file name
However for figshare add-on gives the file path: project/ add-on/ random number/ random number

![screen shot 2015-03-18 at 8 40 55 am](https://cloud.githubusercontent.com/assets/9381062/6709379/56e30a22-cd4f-11e4-92e1-147088e6000a.png)

**Comment by lbanner** [comment]
Wondering if this is just the manner in which figshare notes files: So the file and its location (folder) are designated by seemingly random numbers, and that's what we pull from figshare (?)

**Comment by nicipfeiffer** [comment]
I wondered the same thing, and it may be the case, but I did some investigating.
I checked to see if the random numbers correlate to the figshare file.  The first number does; it is the same number in the file url on figshare.  But the second number I cannot identify.
Also, figshare does support 'projects' for the files to be categorized so I am not sure why that would not be kept with the file in the file path on the OSF.

Looks like @chrisseto worked on showing file path on this PR (closed) maybe he can shed some light?
https://github.com/CenterForOpenScience/osf.io/pull/1760
